src/module.indexer/model/dftx/appointed.oracle.spec.ts

```diff
@@ -0,0 +1,111 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { createTestingApp, stopTestingApp, waitForIndexedHeight } from '@src/e2e.module'
+import { NestFastifyApplication } from '@nestjs/platform-fastify'
+import { OracleMapper } from '@src/module.model/oracle'
+import { OracleHistoryMapper } from '@src/module.model/oracle.history'
+import { OracleTokenCurrencyMapper } from '@src/module.model/oracle.token.currency'
+import { HexEncoder } from '@src/module.model/_hex.encoder'
+import { JsonRpcClient } from '@defichain/jellyfish-api-jsonrpc'
+
+const container = new MasterNodeRegTestContainer()
+let app: NestFastifyApplication
+let client: JsonRpcClient
+
+beforeAll(async () => {
+  await container.start()
+  await container.generate(20)
+  await container.waitForWalletCoinbaseMaturity()
```

Comment:
```suggestion
  await container.waitForWalletCoinbaseMaturity()
```

generate 20 is redundant because waitForWalletCoinbaseMaturity() supersede the block height to 100

Reply:
Haha, okay